docs/en/docs/advanced/additional-responses.md

```diff
@@ -166,7 +166,7 @@ The schemas are referenced to another place inside the OpenAPI schema:
 
 You can use this same `responses` parameter to add different media types for the same main response.
 
-For example, you can add an additional media type of `image/png`, declaring that your *path operation* can return a JSON object (with media type `application/json`) or a PNG image:
+For example, you can add additional media type of `image/png`, declaring that your *path operation* can return a JSON object (with media type `application/json`) or a PNG image:
```

Comment:
This is wrong, original is correct.

Reply:
`add a media..` would make sense here. `additional` is not required. 😅